book/src/design/gadgets/sha256/table16.md

```diff
@@ -192,7 +192,7 @@ For each block $M \in \{0,1\}^{512}$ of the padded message, $64$ words of $32$ b
 are constructed as follows:
 - The first $16$ are obtained by splitting $M$ into $32$-bit blocks $$M = W_0 || W_1 || \cdots || W_{14} || W_{15};$$
 - The remaining $48$ words are constructed using the formula:
-$$W_i = \sigma_1(W_{i-2}) \boxplus W_{i-7} \boxplus \sigma_0(W_{i-15}) \boxplus W_{i-16},$$ for $i = 17, \ldots, 64$.
+$$W_i = \sigma_1(W_{i-2}) \boxplus W_{i-7} \boxplus \sigma_0(W_{i-15}) \boxplus W_{i-16},$$ for $i = 16, \ldots, 63$.
```

Comment:
In Rust, `a..b` is end-exclusive. Is that the same in the book? (I suggest yes)

Alternatively, use the unambiguous notation from the SHA-256 spec `16 <= i <= 63`.

Reply:
I suggest that we use the latter form (or `16 <= i < 64`). That's a form that I could realistically move the protocol spec to using; end-exclusive ranges are not (unfortunately).